### PR TITLE
Fixed #27174 -- Explained where PollsConfig comes from in tutorial 2.

### PR DESCRIPTION
**Pull request by ankur0493**
https://code.djangoproject.com/ticket/27174

### CODE REVIEW

File: docs/intro/tutorial02.txt [hunk 1]
```diff
@@ -204,6 +204,9 @@ But first we need to tell our project that the ``polls`` app is installed.
     you can distribute apps, because they don't have to be tied to a given
     Django installation.
 
+To include the app in our project, we will need to include it's configuration
+file which is present in the :file:`apps` directory inside the app folder.
+The configuration file for our app is present at :file:`polls/apps/PollsConfig.py`
```
Comment (auvipy) [inline docs/intro/tutorial02.txt]: isn't it PollsConfig class in polls/apps.py?

Comment (ankur0493) [inline docs/intro/tutorial02.txt]: Yes it is. Sorry, my bad. I have corrected the same. Please check updated diff.

File: docs/intro/tutorial02.txt [hunk 2]
```diff
@@ -204,8 +204,11 @@ But first we need to tell our project that the ``polls`` app is installed.
     you can distribute apps, because they don't have to be tied to a given
     Django installation.
 
-Edit the :file:`mysite/settings.py` file again, and change the
-:setting:`INSTALLED_APPS` setting to include the string
+To include the app in our project, we will need to include it's configuration
+which is present in the :file:`apps` directory inside the app folder. The
+configuration for our app is present as ``PollsConfig`` in the
+:file:`polls/apps.py`. Edit the :file:`mysite/settings.py` file again, and
```
Comment (berkerpeksag) [inline docs/intro/tutorial02.txt]: in the :file:\`polls/apps.py\` -> in the :file:\`polls/apps.py\` file.

**Comment by berkerpeksag** [comment]
> Do I need to merge this change with 1.9.x and 1.10.x branches?

Backporting documentation changes and bug fixes are usually handled by Django core developers.

File: docs/intro/tutorial02.txt (hunk 2 above)
Comment (ankur0493) [inline docs/intro/tutorial02.txt]: Thanks. Corrected.

**Comment by timgraham** [comment]
merged in 40d501147176352b694ec8b67b09127b20193ad2, thanks!

**Pull request closed without merging**